Box.V2/Models/BoxUserFileCollaborationEventSource.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Newtonsoft.Json;
+
+namespace Box.V2.Models
+{
+    public class BoxUserFileCollaborationEventSource : BoxEntity
+    {
+        public const string FieldFileId = "file_id";
+        public const string FieldFileName = "file_name";
+        public const string FieldUserId = "user_id";
+        public const string FieldUserName = "user_name";
+        public const string FieldParent = "parent";
+
+        [JsonProperty(PropertyName = FieldFileId)]
+        new public string FileId { get; private set; }
```

Comment:
You shouldn't need `new` on any of these properties, since they doesn't collide with any properties in the base class.